src/Terminus.php

```diff
@@ -2,47 +2,86 @@
 
 namespace Pantheon\Terminus;
 
-use Symfony\Component\Console\Application;
-use Symfony\Component\Console\Input\InputOption;
+use Consolidation\AnnotatedCommand\CommandFileDiscovery;
+use Robo\Runner;
 
-class Terminus extends Application
+class Terminus
 {
+    /**
+     * @var string[]
+     */
+    private $args;
+    /**
+     * @var string
+     */
+    private $commands_directory;
+    /**
+     * @var \Robo\Runner
+     */
+    private $runner;
+    /**
+     * @var string
+     */
+    private $top_namespace = 'Pantheon\Terminus\Commands';
 
     /**
-     * @inheritdoc
+     * Runner constructor
      */
     public function __construct()
     {
-        $config = new Config();
-        parent::__construct('Terminus', $config->get('version'));
+        $this->commands_directory = __DIR__ . '/Commands';
+        $this->setRunner();
     }
 
     /**
-     * @inheritdoc
+     * Runs the instantiated Terminus application
+     *
+     * @return integer The exiting status code of the application
      */
-    protected function getDefaultHelperSet()
+    public function run(array $arguments)
     {
-        $helper_set = parent::getDefaultHelperSet();
-        //$helper_set->set(new Pantheon\Terminus\Helpers\FileHelper());
-        return $helper_set;
+        $config = new Config();
```

Comment:
It's only used here and because we're switching to Robo's Runner, we can no longer inject it into our command classes.